.github/workflows/build.yaml

```diff
@@ -0,0 +1,28 @@
+name: Build and test operator images on push to every branch except main and pull requests
+on:
+  pull_request:
+    branches: [main]
+  push:
+    branches-ignore: [main]
+jobs:
+  build-test-operator:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/cache@v2
+        with:
+          path: |
+            ~/.cache/go-build
+            ~/go/pkg/mod
+            ~/go/bin/
+          key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
+          restore-keys: |
+            ${{ runner.os }}-go-
+      - uses: actions/setup-go@v2
+        with:
+          go-version: 1.16
+      - name: Build & Test
+        run: |
```

Comment:
You may wish to encapsulate this in a make file so that you can do things like linting, etc in a centralized place.

Reply:
Yes will be adding a Makefile as well.